module/dkg/broker.go

```diff
@@ -20,7 +23,7 @@ import (
 const RETRY_MAX = 5
 
 // RETRY_MILLISECONDS is the number of milliseconds to wait between retries
-const RETRY_MILLISECONDS = 1000
+const RETRY_MILLISECONDS = 1000 * time.Millisecond
```

Comment:
If it's typed as a duration, we can omit the unit in the name
```suggestion
const RETRY_PERIOD = 1000 * time.Millisecond
```

Reply:
It's more for readability and aesthetics

Reply:
Sure, but it is no longer a variable representing a number of milliseconds. It's a variable representing a number of nanoseconds, with type information that lets it be treated abstractly as a duration without worrying about the underlying unit of measurement. If someone tried to use this value to represent a number of milliseconds, they would get the wrong value!